src/js/construct-element.js

```diff
@@ -11,7 +11,15 @@ export default {
 		}
 
 		messageElement.classList.add('o-message', `o-message--${opts.type}`, 'o-message--closed');
-		if (opts.inner) { messageElement.classList.add('o-message--inner'); }
+		if (!opts.close) {
+			// when close is disabled add the declarative close attribute
+			// which is used to apply style
+			messageElement.setAttribute('data-close', false);
+		}
+
```

Comment:
is this attribute new? should we be prefixing it with `o-message` or anything?

Reply:
it is not a new attribute but I think we should be prefixing! but don't see it in the spec? [related discussion](https://github.com/Financial-Times/origami-website/issues/284)